src/libs/OptionsListUtils.js

```diff
@@ -294,7 +294,7 @@ function isCurrentUser(userDetails) {
 
     // Initial check with currentUserLogin
     let result = currentUserLogin.toLowerCase() === userDetailsLogin.toLowerCase();
-    const {loginList} = currentUser;
+    const loginList = _.isEmpty(currentUser) ? [] : currentUser.loginList;
```

Comment:
Hmm, is the change from `{loginList}` to `loginList` ncessary?

Reply:
Ah. Tests were failing earlier due to this. (currentUser can be undefined & null here)
https://github.com/Expensify/App/pull/4647#issuecomment-899445036

Reply:
I mean specifically that it used to be wrapped in `{ }`

Reply:
Could there have been a reason for that? Wanted to be sure before we change it

Reply:
This was introduced here 4dafbb1ad011fa3204a4bb838cd89b6614deefb .
But I don't think there could be a reason for `loginList` to be used in that way.